src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/InputLocalFinalVariableName.java

```diff
@@ -188,6 +188,27 @@ void veryLong()
           enough talk */
     }
 
+    // check for try-with-resources
+    void method() throws Exception {
+        final String fileName = "Test";
+        try (BufferedReader BR = new BufferedReader(new InputStreamReader(
+                new FileInputStream(fileName), StandardCharsets.UTF_8))) {
+        }
+        finally {
+
+        }
+    }
+
+    void method2() throws Exception {
+        final String fileName = "Test";
+        try (final BufferedReader BR = new BufferedReader(new InputStreamReader(
```

Comment:
Done. However, not sure, if the new case is good enough. Coudn't imagine anything better.